client/src/pages/automation/project/Project.tsx

```diff
@@ -641,12 +630,32 @@ const Project = () => {
                     loading={componentsIsLoading || taskDispatcherDefinitionsLoading || projectWorkflowsLoading}
                 >
                     {componentDefinitions && !!taskDispatcherDefinitions && workflow?.id && (
-                        <ProjectWorkflow
-                            componentDefinitions={componentDefinitions}
-                            projectId={+projectId!}
-                            taskDispatcherDefinitions={taskDispatcherDefinitions}
-                            updateWorkflowMutation={updateWorkflowMutation}
-                        />
+                        <ResizablePanelGroup className="flex-1" direction="vertical">
+                            <ResizablePanel className="relative" defaultSize={65}>
+                                <ProjectWorkflow
+                                    componentDefinitions={componentDefinitions}
+                                    projectId={+projectId!}
+                                    taskDispatcherDefinitions={taskDispatcherDefinitions}
+                                    updateWorkflowMutation={updateWorkflowMutation}
+                                />
+                            </ResizablePanel>
+
+                            <ResizableHandle />
+
+                            <ResizablePanel className="bg-white" defaultSize={0} ref={bottomImperativePanelHandleRef}>
```

Comment:
Ne sviđa mi se ovo imenovanje - `bottomResizablePanelRef` je puno bolje, imperative/declarative nije nešto šta se specificira u imenu (koliko ja znam), bez `Handle` jer nije postavljen `ref ` na `handle` nego na čitav `panel`